util/cron/test-homebrew.bash

```diff
@@ -32,4 +32,3 @@ log_info "Building tarball with version: ${version}"
 source $CWD/common-homebrew.bash
```

Comment:
I see that common-homebrew has all of the actual testing. In my opinion, it would make more sense to replace the rest of this file with common-homebrew.bash. Why?
 * the `common-bla.bash` files usually just set up the environment for some kind of test, not run the test themselves. That way, they can be useful for manually reproducing.
 * there is currently only one homebrew test config, so there's no need to try to share common-homebrew.bash among different tests. (If we need that in the future, I'd rather see other configs call something like run-homebrew-test.bash that isn't named starting with common).

Reply:
@mppf , I will merge all the changes into one file and will send a separate PR.
I will go ahead and merge this to see if it fixes the failure.

Thanks for the review.